src/main/java/elements/BaseElementFacade.java

```diff
@@ -0,0 +1,88 @@
+package elements;
```

Comment:
we should use default formatting settings : https://developer.exoplatform.org/#id-eclipse-settings
( To replace Tabs with 2 spaces )
+ license Header please